services/src/neo4j/migrations/1.25-change-type-of-flow-event-timestamp/rollback.cql

```diff
@@ -0,0 +1,3 @@
+MATCH (n:flow_event) set n.timestamp=apoc.date.toISO8601(n.timestamp)
```

Comment:
I'd take precautions - like "toInteger(n.timestamp) == n.timestamp" in the "where" section.

Reply:
ok. will add